absl/debugging/symbolize_win32.inc

```diff
@@ -45,13 +45,26 @@ void InitializeSymbolizer(const char*) {
   // symbols be loaded. This is the fastest, most efficient way to use
   // the symbol handler.
   SymSetOptions(SYMOPT_DEFERRED_LOADS | SYMOPT_UNDNAME);
-  if (!SymInitialize(process, nullptr, true)) {
-    // GetLastError() returns a Win32 DWORD, but we assign to
-    // unsigned long long to simplify the ABSL_RAW_LOG case below.  The uniform
-    // initialization guarantees this is not a narrowing conversion.
-    const unsigned long long error{GetLastError()};  // NOLINT(runtime/int)
-    ABSL_RAW_LOG(FATAL, "SymInitialize() failed: %llu", error);
+  DWORD syminitialize_error;
+  constexpr int kSymInitializeRetries = 5;
+  for(int i = 0; i < kSymInitializeRetries; i++) {
+    if (SymInitialize(process, nullptr, true)) {
+      return;
+    }
+
+    // SymInitialize can fail sometimes with STATUS_INFO_LENGTH_MISMATCH
```

Comment:
Should we provide more details about the STATUS_INFO_LENGTH_MISMATCH failure?  I believe it happens when a module load occurs during the SymInitialize() call.

Reply:
Added the extra info. Thanks!